docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -79,47 +85,50 @@ with
 :math:`\sigma_j` -- cross-section for :math:`j`-th atom
 
 
-The formula above is valid for *first order quantum events*, e. g., transitions :math:`0 \rightarrow 1` for each phonon. In order to
-reconstruct spectrum one has to also consider higher order quantum events. For second order quantum event one should
-consider transitions :math:`0 \rightarrow 2`, but also simultaneous transitions :math:`0 \rightarrow 1`, :math:`0 \rightarrow 1'` for different phonons. Within harmonic approximation all second
-order transitions form the following  set: :math:`\lbrace \omega_1 + \omega_1, \omega_1 + \omega_2, \omega_1 + \omega_3, \ldots,  \omega_p + \omega_p \rbrace`.
-Cardinality of this set is :math:`p^2`, where :math:`p` is a number of fundamentals. In practice one can reduce this number by taking into consideration realistic energy  window
+The formula above is valid for *first-order quantum events*, i.e. transitions :math:`0 \rightarrow 1` for each phonon. (We neglect events that transfer energy *to* the scattered neutron.)
```

Comment:
is the bit in brackets for the sentence before?

Reply:
How about:
```suggestion
The formula above is valid for *first-order quantum events*, i.e. transitions :math:`0 \rightarrow 1` for each phonon and the transfer of energy *to* the scattered neutrons are neglected.
```

Reply:
I find combing the sentences to be needlessly confusing. We could get more clarity with something like
```rst
The formula above covers the *first-order quantum events* -- specifically the transitions :math:`0 \rightarrow 1` for each phonon.
The :math:`1 \rightarrow 0` events (i.e. energy transfer *to* the scattered neutron) would be infrequent at experimental conditions and are neglected.
```